<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>324</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>324</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>324</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lb_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>311</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/tb/ysxs.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tw_login_register">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_login">
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QLabel" name="lb_tel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>71</width>
       <height>36</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>12</height>
      </size>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le__tel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;手机号为11位数字&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>13333333333</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>71</width>
       <height>36</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>12</height>
      </size>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;密码不多于20位&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>101</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_commit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>101</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_register">
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QLineEdit" name="le__tel_register">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;手机号位11位数字&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password_register">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;密码不多于20位&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tel_register">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>36</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>12</height>
      </size>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear_register">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>101</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password_register">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>36</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>12</height>
      </size>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_commit_register">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>101</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_confirm_register">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>12</height>
      </size>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_confirm_register">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_name_register">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>71</width>
       <height>36</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>12</height>
      </size>
     </property>
     <property name="text">
      <string>昵称</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le__name_register">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;昵称不多于10位&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le__tel_register</tabstop>
  <tabstop>le_password_register</tabstop>
  <tabstop>le_confirm_register</tabstop>
  <tabstop>le__name_register</tabstop>
  <tabstop>pb_clear_register</tabstop>
  <tabstop>pb_commit_register</tabstop>
  <tabstop>le__tel</tabstop>
  <tabstop>le_password</tabstop>
  <tabstop>pb_clear</tabstop>
  <tabstop>pb_commit</tabstop>
  <tabstop>tw_login_register</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
